test/quark-core/QuarkWallet.t.sol

```diff
@@ -592,8 +592,6 @@ contract QuarkWalletTest is Test {
         (uint8 cancelV, bytes32 cancelR, bytes32 cancelS) =
             new SignatureHelper().signOp(alicePrivateKey, aliceWallet, cancelOtherOp);
         vm.resumeGasMetering();
-        vm.expectEmit(true, true, true, true);
-        emit CancelOtherScript.Nop();
```

Comment:
Should we still emit something in the `nop` case? Maybe it's not too useful since the emission of a `Nop()` does not necessarily indicate a nonce was actually cancelled. To track cancellations using the no-op approach, we can probably just track the `ExecuteQuarkScript` event.

Reply:
Yeah, there's a better question buried there, which is, we should track whether or not an execution was replayable in the events. I'll add that.

Reply:
I updated the event to include `isReplayable`, that should be more helpful to processors